examples/depositForBurn.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2024, Circle Internet Financial LTD All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {GeneratedType, Registry} from "@cosmjs/proto-signing";
+
+require("dotenv").config();
+const { DirectSecp256k1HdWallet } = require("@cosmjs/proto-signing")
```

Comment:
Nit for consistency:
```suggestion
require("dotenv").config();
const { DirectSecp256k1HdWallet, Registry, GeneratedType } = require("@cosmjs/proto-signing")
```

Reply:
Changed this all around to use `import` statements instead because there were some issues with the consolidated  `require` statements, lmk what you think!